File: TravellerCreator/ClientApp/src/app/sector-creator/models/rtt-worldgen/rtt-worldgen.hex.ts
import {IHex} from "../interfaces/hex";
import {Coordinates} from "../other/coordinates";
import {RttWorldgenStarSystem} from "./rtt-worldgen.star-system";

export class RttWorldgenHex implements IHex {
  coordinates: Coordinates;
  starSystems: RttWorldgenStarSystem[];
}

File: TravellerCreatorModels/RTTWorldgen/RTTWorldgenHex.cs
using TravellerCreatorModels.Interfaces;
using TravellerCreatorModels.Other;

namespace TravellerCreatorModels.RTTWorldgen;

public class RTTWorldgenHex : IHex
{
    public Coordinates Coordinates { get; set; }
    public List<IStarSystem> StarSystems { get; set; } = new();

    public RTTWorldgenHex(Coordinates coordinates)
    {
        Coordinates = coordinates;
    }
}